reference/concepts/default_arguments.md

```diff
@@ -1,3 +1,5 @@
-A default argument is an argument to a function that a programmer is not required to specify. In most programming languages, functions may take one or more arguments. Usually, each argument must be specified in full (this is the case in the C programming language). Later languages (for example, in C++) allow the programmer to specify default arguments that always have a value, even if one is not specified when calling the function.
+# Default arguments
+
+A default argument is an argument to a [function](./functions.md) that a programmer is not required to specify. In most programming languages, functions may take one or more arguments. Usually, each argument must be specified in full (this is the case in the C programming language). Later languages (for example, in C++) allow the programmer to specify default arguments that always have a value, even if one is not specified when calling the function.
```

Comment:
In reference files we use the `[link][ref-link]` style for links. At the bottom of the document, or at the bottom of a section, you would add: `[ref-link]: ./functions.md`. For consistency, can you change this still?